code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -961,6 +1058,17 @@
 	color = "#C8A5DC"
 	overdose_threshold = 30
 	taste_description = "grossness"
+	conflicting_reagents = list("styptic_powder", "synthflesh", "silver_sulfadiazine", "kelotane", "bicaridine", "omnizine")
```

Comment:
miner sticks contain omnizine and tricordrazine

Reply:
Ok I admit that i was partially wrong about this because omnizine decays at one quarter the rate, 5u tricord will only neutralize 1.25u of omnizine but you- wait a second that means 5u of omnizine is enough to stay for the entire 15u of tricord so actually you take 150 damage minus what salbutamol and mining nanites heal with 1.25u of omnizine leftover

Reply:
So, Goof will literally Kill miners with this PR.

GOOF GET THIS SHIT OUTTA HERE!!

Reply:
Why do miner pens stack cure alls lmao

Ill fix that

Reply:
miner pens stack cure alls because lavaland is killzone